src/betterproto/__init__.py

```diff
@@ -875,6 +874,10 @@ def parse(self: T, data: bytes) -> T:
                 )
 
             current = getattr(self, field_name)
```

Comment:
This should be moved below that if statement

Reply:
The line 879 uses the `current` attribute, so putting that below doesn't make any sense